builtin/mainmenu/pkgmgr.lua

```diff
@@ -17,15 +17,15 @@
 
 --------------------------------------------------------------------------------
 local function get_last_folder(text,count)
-	local parts = text:split(DIR_DELIM)
+	local parts = text:split("/")
```

Comment:
This only works if the engine always provides paths with forward slashes. I've attempted to do this in l_mainmenu, but another option would be

```lua
text:gsub("[/\\]+", "/"):split("/")
```